mongodbatlas/cloud_provider_access.go

```diff
@@ -52,7 +52,12 @@ type AWSIAMRole struct {
 	FeatureUsages              []*FeatureUsage `json:"featureUsages,omitempty"`              // Atlas features this AWS IAM role is linked to.
 	IAMAssumedRoleARN          string          `json:"iamAssumedRoleArn,omitempty"`          // ARN of the IAM Role that Atlas assumes when accessing resources in your AWS account.
 	ProviderName               string          `json:"providerName,omitempty"`               // Name of the cloud provider. Currently limited to AWS.
-	RoleID                     string          `json:"roleId,omitempty"`                     // Unique ID of this role.
+	RoleID                     string          `json:"roleId,omitempty"`                     // Unique 24-hexadecimal digit string that identifies the role.
```

Comment:
the name of this struct doesn't make much sense now

Reply:
good point. Updating it

Reply:
updated

Reply:
That will be breaking change. We need release notes update.

Reply:
we just need to add the label breaking and our tooling takes care of the rest, this client is also a zero release (and forever will be) so we have no backward compatibility guarantee

Reply:
I meant to commit the message to reflect this is breaking change. Good to know about labels - very useful.